torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -561,6 +561,15 @@ TORCH_CUDA_CU_API TensorView* gather(
     const std::vector<int>& strides = {},
     bool trim_out_of_bounds = false);
 
+//! A fused multiply and add operator that will
+//!   lower to mma intrinsics.
+TORCH_CUDA_CU_API TensorView* mma(
```

Comment:
Please add comments on the function parameters.

Reply:
Added. Thanks!